mezzanine/project_template/deploy/supervisord.conf

```diff
@@ -0,0 +1,18 @@
+[unix_http_server]
```

Comment:
Mostly to be able to run `supervisord` and `supervisorctl` as the current user instead of requiring sudo (as a convenience when doing server maintenance). However, I just brought this over from the alternate fabfile and will further explore using the distro default to get rid of this file.